src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const isPartner = person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband`;
+  const partner = person.isMarried ? isPartner : 'I am not married';
```

Comment:
use this condition in html layout in the return. Try to avoid creating unnecessary variables.